API.md

```diff
@@ -0,0 +1,79 @@
+# Communications Template Service Api
+
+This page documents the Communications Template service API endpoints
+
+## Classification Types
+The following endpoints are associated with the classification types that are used to specify which template to use when communicating.
+
+### Create a classification type
+
+* `POST /classificationtypes/GEOGRAPHY`
+
+### Delete a classification type
+
+* `DELETE /classificationtypes/GEOGRAPHY`
+
+### GET a classification type
+
+* `GET /classificationtypes/GEOGRAPHY` will return the classification type or None if it doesn"t exist.
```

Comment:
It might be good to have example responses for the `GET` requests if that's possible, to give an example of what these classifiers look like